src/silo/storage/dictionary.cpp

```diff
@@ -12,6 +12,45 @@
 
 namespace silo {
 
+void Dictionary::updateStringLookup(const ColumnName& column_name, const std::string& value) {
+   if (!string_lookup[column_name].contains(value)) {
+      reverse_string_lookup.at(column_name).push_back(value);
+      string_lookup.at(column_name)[value] = string_lookup.at(column_name).size();
+   }
+}
+
+void Dictionary::updatePangoLineageLookup(
+   const ColumnName& column_name,
+   const PangoLineage& pango_lineage
+) {
+   if (!pango_lineage_lookup.at(column_name).contains(pango_lineage)) {
+      reverse_pango_lineage_lookup.at(column_name).push_back(pango_lineage);
+      pango_lineage_lookup.at(column_name)[pango_lineage] =
+         pango_lineage_lookup.at(column_name).size();
+   }
+}
+
+Dictionary::Dictionary(
+   const std::vector<ColumnName>& string_column_names,
+   const std::vector<ColumnName>& pango_lineage_column_names
+) {
+   for (const auto& column_name : string_column_names) {
+      string_lookup[column_name] = std::unordered_map<std::string, ValueId>();
+      reverse_string_lookup[column_name] = std::vector<std::string>();
+   }
+
+   for (const auto& column_name : pango_lineage_column_names) {
+      pango_lineage_lookup[column_name] = std::unordered_map<PangoLineage, ValueId>();
+      reverse_pango_lineage_lookup[column_name] = std::vector<PangoLineage>();
+   }
+}
+
+Dictionary::Dictionary()
+    : Dictionary(
+         std::vector<std::string>({"country", "region", "division", "column"}),
```

Comment:
This looks like a leftover?
```suggestion
         std::vector<std::string>({"country", "region", "division"}),
```